### ISSUE TITLE
interact() starts with pixel-level image zoomed out

### ISSUE THREAD

**Issue by JeffLCoughlin** [KeplerGO/lightkurve#215]
#### Problem description
When starting interact(), the pixel-level image displayed on the right side is zoomed out. The actual image only takes up about 1/4 of the area, and I have to manually go and zoom.

#### Example
```jupyter notebook
from lightkurve import KeplerTargetPixelFile
tpf = KeplerTargetPixelFile.from_archive(210957318)
tpf.interact()
```

In this example, the default scaling is to range from 40 to 70 on the x-axis, and 755 to 781 on the y-axis, while the TPF itself is only 43--51 in x and 756--768 in y.

#### Expected behavior
Ideally interact() would default to having the displayed image zoom set such that the size of the given TPF to maximizes screen usage. In the example above, the xrange and yrange I think should default to the pixel size of the TPF.

#### Environment:
Linux
1.0b13
pip

**Comment by barentsen** [comment]
Confirm. I have noticed this behavior as well. Assigning to @gully to either resolve or declare as non-fixable.

**Comment by gully** [comment]
This issue is resolved in the next version of `interact()` PR #216
